pom.xml

```diff
@@ -234,6 +234,13 @@
                     <groupId>org.apache.maven.plugins</groupId>
                     <artifactId>maven-surefire-plugin</artifactId>
                     <version>2.20.1</version>
```

Comment:
Hello @lukaszlenart .
Since I didn't know if there was some version dependency for the Jenkins/Travis-CI build process I was afraid to mess with those versions (and mess with the PR).  Maybe the maven-surefire-plugin should be updated separately ?
It looks like the most current 2.x maven-surefire-plugin is 2.22.1 (and that 3.0.0-M1 is a new stable build series).

Reply:
👍